mitosheet/mitosheet/types.py

```diff
@@ -344,6 +344,11 @@ class AITransformFrontendResult(TypedDict):
     ]
     ParamValue = str
 
+    class ParamData(TypedDict):
```

Comment:
Document this a bit -- what its' for, why it exists, etc. This is going to end up being an internal interface. 

Also, can we think of a better name here? `ParamData` might be used elsewhere, and if not it's very close to other ones.